website/README.md

```diff
@@ -1,6 +1,6 @@
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-Note: `yarn` is required, otherwise some plugins don't work
+Note: `yarn` is required otherwise, some plugins don't work
```

Comment:
is this really correct?

Reply:
yes, I Checked with a Grammarly tool

Reply:
http://www.centralsoutherntierraen.org/protected/documents/comma_rules2.pdf

you could check rule 10 in the Comma_Rules